front/src/WebRtc/JitsiFactory.ts

```diff
@@ -48,13 +48,35 @@ class JitsiFactory {
             }
             this.jitsiApi = new window.JitsiMeetExternalAPI(domain, options);
             this.jitsiApi.executeCommand('displayName', playerName);
+
+            this.jitsiApi.addListener('audioMuteStatusChanged', this.onAudioChange.bind(this));
+            this.jitsiApi.addListener('videoMuteStatusChanged', this.onVideoChange.bind(this));
         }));
     }
 
     public stop(): void {
+        this.jitsiApi.removeListener('audioMuteStatusChanged', this.onAudioChange.bind(this));
+        this.jitsiApi.removeListener('videoMuteStatusChanged', this.onVideoChange.bind(this));
```

Comment:
Je pense que le removeListener ne va pas marcher.

`this.onVideoChange.bind(this)` va retourner une instance de fonction différente de celle retournée lorsque tu fais le "add". Du coup, tu ne retires rien. Il faut probablement sauvegarder la fonction "bindée" dans une variable de la classe que tu utilises ensuite dans le addListener et dans le removeListener (je fais ça à plusieurs endroits déjà)